.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2.1
 defaults: &defaults
   working_directory: ~/osmosis
   docker:
-    - image: circleci/node:14.17.0
+    - image: cimg/node:18.15.0
```

Comment:
Probably zero benefit if not just negative impact in this use case, but I saw Circle supports ARM for docker and thought that was real neato.

https://discuss.circleci.com/t/product-launch-arm-docker-generally-available/48601

Reply:
oh interesting. maybe something to think about in the future